manifests/s/ServerData/Elevate/2.15.15/ServerData.Elevate.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ServerData.Elevate
+PackageVersion: 2.15.15
+PackageLocale: en-US
+Publisher: ServerData
+PublisherUrl: https://cp.serverdata.net/Portal/User/Login
+PackageName: Elevate Desktop App

```

```suggestion
PackageName: Elevate UC
```
This should match what is written to control panel